app/utilities/schema.py

```diff
@@ -98,14 +98,15 @@ def get_allowed_languages(schema_name, launch_language):
     return [DEFAULT_LANGUAGE_CODE]
 
 
-def load_schema_from_metadata(metadata):
+def load_schema_from_metadata(metadata, language_code=None):
```

Comment:
Enforcing keyword arguments would improve readability?
```suggestion
def load_schema_from_metadata(*, metadata, language_code=None):
```
This will break all the invocations of `load_schema_from_metadata` but easy to fix.

Reply:
Added this 👍